blog/blog_server/server.go

```diff
@@ -84,7 +84,7 @@ func (*server) ReadBlog(ctx context.Context, req *blogpb.ReadBlogRequest) (*blog
 
 	// create an empty struct
 	data := &blogItem{}
-	filter := bson.NewDocument(bson.EC.ObjectID("_id", oid))
+	filter := bson.M{"_id": oid}
```

Comment:
why not **filter := primitive.M{"_id": oid}** instead? bson.M is from another package.